### PR TITLE
Fix pow() return value section

### PR DESCRIPTION
**Pull request by staabm**
I don't think `pow()` can return an `object`?

### CODE REVIEW

**Comment by devnexen** [comment]
Seems you re correct, i do not see any other possibility than float/int.

**Review by devnexen — APPROVED** [review]

**Comment by cmb69** [comment]
Well, https://github.com/php/php-src/blob/d55ef3f339a6d8206ee293d5cc7d9ab6044d3e6a/ext/standard/basic_functions.stub.php#L3212C3-L3212C63 says it can; if that's wrong, the stubs would need to be fixed, and later synced with the docs. If it's not wrong, there should be some clarification in the docs (I think it's about internal objects).

**Comment by devnexen** [comment]
I admit I did not look the stub but just the implementation(s).

**Comment by devnexen** [comment]
Finally I retract, not only GMP overrides but also libraries php decimal do override the do_operation handler.

**Comment by staabm** [comment]
added a paragraph and a example

**Review by cmb69 — APPROVED** [review]
Thank you, @staabm! This looks good to me.

File: reference/math/functions/pow.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
    otherwise it will be returned as a <type>float</type>.
   </para>
   <para>
-   PHP-Extensions may override the behaviour of this operation and make it return objects.
+   PHP-Extensions may override the behaviour of this operation and make it return a object.
```
Comment (devnexen) [inline reference/math/functions/pow.xml:57]: nit: `an object` I think

**Comment by devnexen** [comment]
Thanks !

**Pull request merged**